parser.y

```diff
@@ -10771,4 +10786,95 @@ LoadStatsStmt:
 		}
 	}
 
+/********************************************************************************************
+ *
+ *  Create Sequence Statement
+ *
+ *  Example:
+ *      CREATE SEQUENCE IF NOT EXIST my_sequence
+ *          [INCREMENT BY <increment>]
+ *          [START WITH <start>]
+ *	    [MINVALUE <minvalue> | NOMINVALUE]
+ *          [MAXVALUE <maxvalue> | NOMAXVALUE]
+ *          [CACHE <cache>]
+ *          [CYCLE | NO CYCLE]
+ *          [ORDER | NO ORDER]
+ ********************************************************************************************/
+
+CreateSequenceStmt:
+	"CREATE" "SEQUENCE" IfNotExists TableName CreateSequenceOptionListOpt
```

Comment:
Add an `OptTemporary` here since MariaDB allows creating temporary sequences.
```suggestion
	"CREATE" OptTemporary "SEQUENCE" IfNotExists TableName CreateSequenceOptionListOpt
```

Also, MariaDB also allows all table options to be used here, e.g.

```sql
CREATE SEQUENCE xyz START WITH 41 COMMENT 'this is a sequence'
```

so perhaps include `CreateTableOptionListOpt` at the end as well.

Reply:
Thanks kennytm, that does a great help to me!